### DISCUSSION TITLE
Enable ADA price value in real time, like other wallets do

### DISCUSSION THREAD

**Discussion by lluishi93** [Emurgo/yoroi-mobile#1876] [Ideas]
I'd be nice to enable a feature, so the user can see ADA's real price in real time, like other wallets like Exodus or Coinbase do.
![image](https://user-images.githubusercontent.com/14786440/108822834-ee2c0e00-75bf-11eb-9ebf-59eab11bdac9.png)

**Comment by v-almonacid** [comment]
Indeed, this is part of our roadmap.

**Comment by lluishi93** [comment]
@v-almonacid  nice! is there any prevision on when this feature is going to be enabled?

**Comment by v-almonacid** [comment]
Can't say exactly but hopefully this quarter 🤞

**Comment by lluishi93** [comment]
Any news on this...?

**Comment by X4** [comment]
Bump

**Comment by stackchain** [comment]
going to pro on `4.8.0`
